k8s/elasticsearch/Makefile

```diff
@@ -82,12 +83,13 @@ app/build:: .build/elasticsearch/elasticsearch \
 .build/elasticsearch/ubuntu16_04: .build/var/REGISTRY \
                                   .build/var/TAG \
                                   | .build/elasticsearch
-	docker pull marketplace.gcr.io/google/ubuntu1604
-	docker tag marketplace.gcr.io/google/ubuntu1604 \
+	docker pull marketplace.gcr.io/google/ubuntu1604:latest
```

Comment:
Currently it is already pulling the latest image - why did you prefer to add the explicit `latest` tag?

Reply:
The formatted Makefiles are generated semi-automatically. do you think we should omit `:latest` tag in this case?